openstates/in/bills.py

```diff
@@ -45,198 +45,185 @@ class INBillScraper(BillScraper):
+            if not title:
+                title = bill_id
+                self.logger.warning("Bill is missing a title, using bill id instead.")
+
+
+            original_chamber = "lower" if bill_json["originChamber"].lower() == "house" else "upper"
+            bill = Bill(session,original_chamber,bill_id,title)
+            
+            bill.add_source(api_source,note="API key needed")
+            bill.add_source(self.make_html_source(session,bill_id))
+            #documents/versions
+            #todo - these are terrible.
+
+
+            #sponsors
+            positions = {"Representative":"lower","Senator":"upper"}
+            for s in bill_json["authors"]:
+                bill.add_sponsor("primary",
+                    self.get_name(s),
+                    chamber=positions[s["position_title"]],
+                    official_type="author")
+
+            for s in bill_json["coauthors"]:
+                bill.add_sponsor("cosponsor",
+                    self.get_name(s),
+                    chamber=positions[s["position_title"]],
+                    official_type="coauthor")
+
+            for s in bill_json["sponsors"]:
+                bill.add_sponsor("primary",
+                    self.get_name(s),
+                    chamber=positions[s["position_title"]],
+                    official_type="sponsor")
+
+            for s in bill_json["cosponsors"]:
+                bill.add_sponsor("cosponsor",
+                    self.get_name(s),
+                    chamber=positions[s["position_title"]],
+                    official_type="cosponsor")
+
+            #actions
+            action_link = bill_json["actions"]["link"]
+            api_source = api_base_url + action_link
+            actions = client.get("bill_actions",session=session,bill_id=bill_id.lower())
+            for a in actions["items"]:
+                action_desc = a["description"]
+                chamber = "lower" if a["chamber"]["name"].lower() == "house" else "upper"
+                date = a["date"]
+                date = datetime.datetime.strptime(date,"%Y-%m-%dT%H:%M:%S")
+                
+                action_type = []
+                d = action_desc.lower()
+                committee = None
+                if "first reading" in d:
+                    action_type.append("bill:reading:1")
+
+                if ("second reading" in d
+                    or "reread second time"):
+                    action_type.append("bill:reading:2")
+
+                if "third reading" in d:
+                    action_type.append("bill:reading:3")
+                    if "passed" in d:
+                        action_type.append("bill:passed")
+                    if "failed" in d:
+                        action_type.append("bill:failed")
+
+                if ("referred" in d and "committee on" in d
+                    or "reassigned" in d and "committee on" in d):
+                    committee = d.split("committee on")[-1].strip()
```

Comment:
I'm not 100% that the "committee on" is consistently cased (or won't change someday, given other data consistency issues). So if Billy can handle potentially different casing for committees in the matching, I think it'd be safer to leave it this way. @paultag can you confirm? Otherwise I'll make the change.